### ISSUE TITLE
Application Attach Failing

### ISSUE THREAD

**Issue by iShorty123** [Roemer/FlaUI#94]
Hello, when I do an Application.Attach("Executable Name Here") I get the following error:
"System.ComponentModel.Win32Exception:'A 32 bit processes cannot access modules of a 64 bit process.'"

This error is triggered when this line of code is ran in Application.cs:
"Logger.Default.Debug("[Attaching to process:{0}] [Process name:{1}] [Process full path:{2}]", process.Id, process.ProcessName, process.MainModule.FileName);"

It is the process.MainModule.FileName (or just process.MainModule) that triggers the error. 

Through experimentation I've discovered that if I comment out this logger line, then the process still attaches correctly and works with FlaUI. 

Screen scrape of error:
![capture](https://user-images.githubusercontent.com/24724723/30507117-9ca47a96-9a3e-11e7-8688-a7c97cc8b6a2.PNG)

**Comment by Roemer** [comment]
What methods do you use from the Application class?
It seems to me that your application is 64 bit but your test runs in 32 bit. Can you try running the test in 64 bit?

**Comment by iShorty123** [comment]
A lot of different ones I believe. Would tell you which ones specifically, but won't be back to work till Monday. The main thing we do is assign an app with attach or some form of attach. It has so far worked with the 3 other application that we have used it for. It's just the 4th one that it is giving us this error.

I can let you know now though that the application is being built in debug mode, for any CPU. I tried switch it to x86 and x64 modes, but neither made a difference on the error. 

I know when I ran Environment.is64BitProcess it returned false, once again, for both x86 and x64, as well as Any CPU.
